index.html

```diff
@@ -5,7 +5,7 @@
     <meta charset="utf-8" />
     <meta content="width=device-width, initial-scale=1.0" name="viewport" />
 
-    <title>Restaurantly Bootstrap Template - Index</title>
+    <title>A home of delicous cakes</title>
```

Comment:
1. We need to fix the spelling of delicious.
2. We need to add Bubby Bakery as a prefix to this line. 'Bubby Bakery - A home of delicious cakes'

Reply:
ok ill go though it